### ISSUE TITLE
Implement meter

### ISSUE THREAD

**Issue by ryanccarelli** [ryanccarelli/ssljax#34]
Task.meter takes logits, targets, weights and returns metrics
